### PR TITLE
adding luningwang's coordinants

### LINKED ISSUE #12177: Please add me
Hello! :wave: Please add me as a collaborator to this repo. 

First day using GitHub!

### PR DESCRIPTION
**Pull request by luningwang**
@githubteacher adding pin of my location.

This pull request closes #12177.

### CODE REVIEW

**Comment by WalterSmith12** [comment]
Keep up  the good work @luningwang

**Comment by tarains** [comment]
Looks like what I did, good job

**Comment by lindayonemoto** [comment]
Nicely done! :+1:

File: _pins/luningwang.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+GitHubHandle: luningwang
```
Comment (crichID) [inline _pins/luningwang.json]: @luningwang you are very close. yaml is very picky and requires that the capitalization is exact. You will need to:

- Change `GitHubHandle` to `githubHandle`
- Change `Latitude` to `latitude`

After you fix these two things,  your tests should pass.  Here is a video to help you make the changes if needed:   https://vimeo.com/201011288

**Comment by USA2021** [comment]
@luningwang it looks cool

**Comment by hollenberry** [comment]
@luningwang  I went ahead and fixed the syntax issues recommended by Cynthia above in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/12185/commits/2ce27537aff3b00576db40e752e479f9bbdadb77 and https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/12185/commits/39d1160fa95c0f433dfe2a892e21bf0cca0817d. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by December 15th, I will go ahead and do it just to keep the repository clean!

**Pull request merged**